data/json/professions.json

```diff
@@ -5137,6 +5137,143 @@
                 "bra",
                 "panties"
             ]
+        },
+            {
+        "type": "profession",
+        "ident": "escape_con",
+        "name": "Escaped Prisioner",
```

Comment:
We already have a convict profession. The stats are different than yours but from a RP prospective its a repeat.

Reply:
It's supposed to be supporting for a different scenario.

Reply:
See latest commit.

Reply:
Which scenario? All you have in this PR is professions.

Reply:
I am still working on the scenarios.